src/Pages/AddItem.js

```diff
@@ -1,9 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
-const AddItem = () => (
-  <div>
-    <h1 className="title is-1">Add Item </h1>
-  </div>
-);
+export default function AddItem({ token }) {
+  const [item, setItem] = useState('');
+  const [urgency, setUrgency] = useState(7);
 
-export default AddItem;
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    e.target.reset();
+    db.collection(token).add({
+      item,
+      urgency,
+      lastPurchased: null,
+    });
+    // resets input field after submission
+    setUrgency(7);
+    setItem('');
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="item">
+          <b>Grocery item:</b>
+        </label>{' '}
+        <br />
+        <input
+          type="text"
+          id="item"
+          value={item}
+          required={true}
+          onChange={(e) => setItem(e.target.value)}
+        />{' '}
+        <br />
+        <br />
+        {/* This fieldset is similar to what we wad going on with the div, just better for accessibility */}
+        <fieldset onChange={(e) => setUrgency(parseInt(e.target.value))}>
```

Comment:
I have some ideas around improving this logic, but I think it's best saved for office hours. If you or are remember this comment come tomorrow, let's talk about it 🤠

Reply:
@tannaurus Hopefully I remember, my memory is not the best.